### DISCUSSION TITLE
HyperHDR LED Device not being Recognized.

### DISCUSSION THREAD

**Discussion by DSELCTD** [awawa-dev/HyperHDR#677] [Q&A]
Hey! I am trying to do my first raspberry pi 3b+ build and I wanted to start with these ambilight set ups as I don't want to spend $500 on Hue Sync Setup. I have followed everything point for point on various videos, forums, and websites and for whatever reason my LED lights will not be recognized by the HyperHDR software. I am using a 5V 10A PSU that was recommended, I counted LEDs and calculated my power usage just in case and it all added up. It reads my USB Device and my capture device but will not read my LEDs, I am connecting the data line to the GPIO 1 pin that is PWM_CLK also known as the GPIO 18 as described in by other users. Do you guys have any fix for this? I could really use the help, thought this thread was the best place to ask. Thanks! I am looking forward to using this program and further learning the systems involved.

**Comment by awawa-dev** [comment]
Hi
Not many details about the hardware used or the HyperHDR configuration/logs, but it looks like you chose a more difficult one and, in addition, an officially unsupported solution, as HyperHDR certainly informed you about in the driver configuration. Additionally, controlling neopixel LEDs (ws2812b, sk6812 etc) directly from the RPi does not currently work on the latest RPi5 due to serious hardware changes in this board.

**Reply by DSELCTD** [reply]
I am not using the RPi5, I am using the RPi3B+. Do I need a voltage step up for my data line?

**Reply by DSELCTD** [reply]
I am using WS2812 LEDs aswell. Some users have good luck with these but I have not had any luck with it being read by HyperHDR. There is no data logs as nothing is attempting to connect.

**Reply by awawa-dev** [reply]
>  Do I need a voltage step up for my data line?  

Certainly you need it if you want to do it correctly, according to LED technical specifications. However, there are other reasons that may cause problems with proper signal generation on the RPi and Neopixels need perfect timings.

> There is no data logs as nothing is attempting to connect.

Upload the FULL HyperHDR logs taken 2-3 minutes after launch (it is best to upload them to pastebin and provide a link).
